contracts/SeriesIndexer.sol

```diff
@@ -0,0 +1,448 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol" as Ownable;
+
+contract SeriesIndexer is Ownable.Ownable {
+    // Counter
+    uint256 private nextSeriesID = 1;
+    uint256 private nextArtistID = 1;
+
+    struct Series {
+        string metadata;
```

Comment:
I suppose this field is for series metadata source, so it should store the uri so the name should be `metadataURI`